optuna/visualization/_optimization_history.py

```diff
@@ -98,7 +98,7 @@ def _get_optimization_history_plot(
 
     layout = go.Layout(
         title="Optimization History Plot",
-        xaxis={"title": "#Trials"},
+        xaxis={"title": "trial number"},
```

Comment:
While it can be argued, using capitals is likely more consistent, with e.g. `"Objective Value"` and other labels and such in https://optuna.readthedocs.io/en/latest/reference/visualization/index.html.
```suggestion
        xaxis={"title": "Trial Number"},
```

Reply:
No worries, thanks for the feedback! Made new commits with the suggested changes!

Reply:
Thanks a lot, looks great!